examples/asr/conf/conformer/fastconformer/fastconformer_transducer_bpe.yaml

```diff
@@ -0,0 +1,259 @@
+# It contains the default values for training a Conformer-Transducer ASR model, large size (~120M) with Transducer loss and sub-word encoding.
+
+# Architecture and training config:
```

Comment:
We'll also need multilingual configs etc, we can add them in the future I suppose.

Reply:
right once we train..